test/cli_exec_test.go

```diff
@@ -137,7 +139,33 @@ func (suite *PouchExecSuite) TestExecStoppedContainer(c *check.C) {
 
 // TestExecInteractive test "-i" option works.
 func (suite *PouchExecSuite) TestExecInteractive(c *check.C) {
-	//TODO
+	name := "TestExecInteractive"
+	res := command.PouchRun("run", "-d", "--name", name, busyboxImage, "sleep", "100000")
```

Comment:
Since `res` not used by other code, how about `command.PouchRun("run", "-d", "--name", name, busyboxImage, "sleep", "100000").Assert(c, icmd.Success)`

Reply:
updated.